rust/agama-lib/share/examples/profile.jsonnet

```diff
@@ -1,4 +1,18 @@
+// This is a Jsonnet file. Please, check https://jsonnet.org/ for more
+// information about the language.
+// For the schema, see
+// https://github.com/openSUSE/agama/blob/master/rust/agama-lib/share/profile.schema.json
+
+// The "hw.libsonnet" file contains hardware information of the storage devices
+// from the "lshw" tool. It is generated by running as root:
```

Comment:
I think the most important information is that it is explicitly run by Agama itself, there is no magic for that.
https://github.com/openSUSE/agama/blob/b4bf25fdffd46735800a8396f1715fffe181e1db/rust/agama-lib/src/profile.rs#L128

Reply:
I merged too soon. I opened https://github.com/openSUSE/agama/pull/655.